Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -66,28 +66,32 @@ class CurrentUserUpdater: Worker {
         currentUserId: UserId,
         completion: ((Error?) -> Void)? = nil
     ) {
-        database.write { (session) in
+        database.write({ session in
```

Comment:
My tests failed because I was relying on database to update before this function returns. But here we did not use DB write's completion handler.